tests/blank-trailing-space.cpp

```diff
@@ -1,13 +1,13 @@
-// This comment includes trailing space. 
+// This comment includes trailing space.
```

Comment:
Lies!  Trailling spaces throughout this file have been mistakenly eliminated.

Reply:
Trailing spaces (on an otherwise non-blank line) have been intentionally eliminated per the original request. Trailing spaces on an otherwise blank line have been preserved.